code/game/atoms.dm

```diff
@@ -860,3 +862,21 @@ Proc for attack log creation, because really why not
 /atom/proc/reset_light()
 	turn_light(null, TRUE)
 
+/**
+  * Recursive getter method to return a list of all ghosts orbitting this atom
+  *
+  * This will work fine without manually passing arguments.
+  */
```

Comment:
```suggestion
 * Recursive getter method to return a list of all ghosts orbitting this atom
 *
 * This will work fine without manually passing arguments.
 */
```